### DISCUSSION TITLE
Modelling 1D 2 Phase flow

### DISCUSSION THREAD

**Discussion by Jake-HW-Miles** [idaholab/moose#28067] [Q&A Modules: Thermal Hydraulics]
Hi there, 
In my pursuit of modelling power conversion systems in MOOSE, I am curious to know if any of the 2 phase modelling features from RELAP made it into MOOSE? 

For reference, I am aiming for something similar to the attached image:
![image](https://github.com/idaholab/moose/assets/156903724/1727fc2c-07e9-40c0-a6b7-043d65908d55)

Thanks! :)

**Comment by GiudGiud** [comment]
Hello

none of the two phase discretization did make it into the thermal hyrdraulics module

The two phase fluid properties are in the fluid properties module and the submodules we are adding 

Guillaume

**Reply by Jake-HW-Miles** [reply]
Great, thank you! I think `TwoPhaseFluidPropertiesIndependent` will do at this stage!

**Reply by GiudGiud** [reply]
You can apply for a relap7 license at the INL NCRC website
